### DISCUSSION TITLE
permission pb with material files?

### DISCUSSION THREAD

**Discussion by mrmagicm93** [community/community#127411] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hello, I seems having pb with "Material file":
I'm on root android box, but When I tried to move files from the root directory to the main storage or usb drive, it doesn't seems to work. Also, I cannot seem to copy files from one directoy on the usb to another directory on the usb key. Are those bugs of Material file or does Majisk doesn't grant total permissions?

**Comment by ebndev** [comment]
Hi @mrmagicm93,

Thanks for being a part of the GitHub Community, we're glad you're here!

The Profile category is for conversations related to your GitHub profile page. I have moved this post to General, however, it looks like you're looking for support from a specific. You might want to try asking for help somewhere that focuses on this project, such as opening an issue in the project's repository or seeing if they have a support page.

We want to make sure you’re getting the best support you can and that your message goes where you intended, but this may not be the right place.

Best of luck!

P.S would you mind telling us how you ended up reaching out for support in the Profile Category of Discussions? We are getting a lot of posts where folks are reaching out for support from projects hosted on GitHub and we want to ensure there aren't links or directions out there that may be misguiding folks.
